app/lib/efile/relationships.yml

```diff
@@ -3,66 +3,88 @@ relationships:
   - id: daughter
     relationship: qualifying_child
     irs_enum: DAUGHTER
+    skip_relative_household_test: true
```

Comment:
i added this here because it kind of reflects the format that IRS provides this data to us in which is each relationship either needs this test or not. however, should i invert the logic? relative_household_test: false (i invert the logic in the relationship.rb that consumes this file.)